README.md

```diff
@@ -51,27 +51,58 @@ try {
 ```
 ## Reference
 ### Methods
-`startUpdateFlow()`
+#### startUpdateFlow(appUpdateType, clientVersionStalenessDays)
```

Comment:
Hi, @ybbond while it's nice to have the function input on the `h4` i much prefer the complete function with inputs on the code example itself. 

For reference https://reactnative.dev/docs/appregistry#registercomponent

Reply:
I have updated the section title for that method, omitting the params

Reply:
LGTM